src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -565,22 +559,6 @@ protected virtual void Dispose(DisposeTypes type)
                 Log.Error("NUI", $"[ERR] SwigCPtr is NULL, need to check! me:{me}");
             }
 
-            if (swigCPtrCopy.Handle != global::System.IntPtr.Zero)
-            {
-                swigCMemOwn = false;
-                var nativeSwigCPtr = swigCPtrCopy.Handle;
-
-                // Remove swigCPtrCopy first. Now HasBody() return false.
```

Comment:
// Remove swigCPtrCopy first. Now HasBody() return false.

To make `HasBody()` return false during object destruction signal, I think we'd better implement something like this.
(Even it looks not beautiful)
`
                var nativeSwigCPtr = swigCPtr.Handle;
                swigCPtr = new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero);
                if (swigCMemOwn)
                {
                    swigCMemOwn = false;
                    ReleaseSwigCPtr(new global::System.Runtime.InteropServices.HandleRef(this, nativeSwigCPtr));
                }
`

Reply:
Thank you very much
I have changed. please review again.